### ISSUE TITLE
Content Model Debt - titles and standfirst tidy up

### ISSUE THREAD

**Issue by andygnewman** [Financial-Times/next-es-interface#527]
### Proposed future data model

CAPI V2 has tidies up the titles and standfirst to the following model

```
    title: "Cabinet split over plan to put workers on UK company boards",
    alternativeTitles: {
        promotionalTitle: "Cabinet split on plan to put workers on UK boards"
    },
    standfirst: "Senior minister warns ‘it is not going to happen’"
```

Current ES model is;

```
    title: "Cabinet split over plan to put workers on UK company boards",
    titles: [
        "Cabinet split over plan to put workers on UK company boards",
        "Cabinet split on plan to put workers on UK boards"
    ],
    summaries: [
        "Senior minister warns ‘it is not going to happen’"
    ],
    standfirst: "Senior minister warns ‘it is not going to happen’",
```

Proposed to just retain the CAPI V2 properties of title, alternativeTitles, standfirst.

Deprecate titles, summaries.
### What to do when no standfirst?

Not all content has a standfirst, eg. FastFT, Podcasts, Photo Diary entries etc. 
In these circumstances when we have prepped the data for a card implementation we have created an 'alternative' standfirst / subheading from either the first `<p>` of the openingHTML or from the bodyHTML.

Proposed solution:
Use this logic to create a `subheading` property

**Comment by andygnewman** [comment]
Steps to achieve this;

## 1. Add alternativeTitles and subheading properties to article model
- [ ]  Put PR here

## 2. Wait for all content to be reingested
- [ ] Approx 8/9 days after step 1 merged in.

## 3. Deprecate existing uses of titles, summaries

## 4. Remove titles and summaries from article model

**Comment by i-like-robots** [comment]
Do we need both a standfirst and a subheading? I'm not sure I understand the difference (other than how a subheading may be derived from backup sources).

**Comment by i-like-robots** [comment]
Fixed by #690

**Comment by i-like-robots** [comment]
https://github.com/Financial-Times/google-amp/pull/213
